server/cluster/shard.go

```diff
@@ -21,3 +24,57 @@ type ShardTablesWithRole struct {
 	tables    []*Table
 	version   uint64
 }
+
+// fsm const
```

Comment:
It is better to keep any comment as a statement rather than just some broken words.